source/uwp/Renderer/lib/ActionHelpers.cpp

```diff
@@ -0,0 +1,1034 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+#include "pch.h"
+
+#include "ActionHelpers.h"
+#include "AdaptiveImage.h"
+#include "AdaptiveRenderArgs.h"
+#include "AdaptiveShowCardActionRenderer.h"
+#include "LinkButton.h"
+
+using namespace Microsoft::WRL;
```

Comment:
This file is part of the xamlbuilder refactor and shouldn't be coming in, right?

Reply:
indeed. I'll have it fixed shortly :)

Reply:
fixed with 6fccd7d

Reply:
and verified changes work in the visualizer